src/screens/SavedEventListScreen/NoSavedEvents.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import React, { PureComponent } from "react";
+import { Image, StyleSheet, View } from "react-native";
+import type { NavigationScreenProp, NavigationState } from "react-navigation";
+import Text from "../../components/Text";
+import ContentPadding from "../../components/ContentPadding";
+import ButtonPrimary from "../../components/ButtonPrimary";
+import whiteCheck from "../../../assets/images/whiteCheck.png";
+import iconSave from "../../../assets/images/save.png";
+import noSavedEvents from "../../../assets/images/howToSaveEvents.png";
+import text from "../../constants/text";
+import { cardBgColor, titleTextColor } from "../../constants/colors";
+import { EVENT_LIST } from "../../constants/routes";
+import strings from "../../constants/strings";
+
+type Props = {
+  navigation: NavigationScreenProp<NavigationState>
+};
+
+class NoSavedEvents extends PureComponent<Props> {
+  eventList = () => {
+    this.props.navigation.navigate(EVENT_LIST, {
+      title: strings.featuredEventsTitle
+    });
+  };
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <Image style={styles.image} source={noSavedEvents} />
+        <View style={styles.contentOverlay}>
+          <Text type="h1" style={styles.title}>
+            {text.noSavedEventsTitle}
+          </Text>
+          <Text style={styles.infoText}>
+            {text.noSavedEventsPart1}&nbsp;
+            <Image
+              accessibilityLabel="'save this event' button"
+              source={iconSave}
+            />
+            &nbsp;{text.noSavedEventsPart2}
+          </Text>
+          <ButtonPrimary onPress={this.eventList}>
+            {text.noSavedEventsButton}
+          </ButtonPrimary>
+        </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 3,
+    flexDirection: "column",
+    flexWrap: "wrap",
+    marginHorizontal: 12
+  },
+  image: {
+    alignSelf: "center"
+  },
+  title: {
+    color: titleTextColor,
+    textAlign: "center",
+    paddingBottom: 12
```

Comment:
We have a column component that can do this kind of spacing for you. Might be worth checking out to avoid many paddings.

Reply:
thanks, I'll have a look!

Reply:
yes, that made things easier, thanks @RGBboy !